### PR TITLE
Term entry swiftui frame

### LINKED ISSUE #1329: [Term Entry] SwiftUI ViewModifier: `.frame()`
### Reason/inspiration (optional)

Please write a new entry under:  content/swiftui/concepts/viewmodifier/terms/ about the modifier `.frame()` in Swift.

- A "Syntax" section with pseudo code snippet. [Include height and width]
- An "Example" section with a swift code snippet and image to accompany. Images should go in the `media/` folder.

To get started with the PR and entry file:

Under content/swiftui/concepts/viewmodifier/terms/, create a new folder called `frame`. Then create a new file called `frame.md` under content/swiftui/concepts/viewmodifier/terms/`frame`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new entry for SwiftUI ViewModifier: .frame()
- Closes #1329 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### Additional Information
I am using npm; I have node 18.12.
I was unable to run format:verify (prettier is not recognized)
I was unable to run lint:md (markdownlint is not recognized)
I was unable to run test (jest is not recognized)

### CODE REVIEW

File: content/swiftui/concepts/viewmodifier/terms/frame/frame.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+### Defining a fixed frame
+
+```pseudo
+Text("Some text is here.")
+    .frame(width: x, height: y, alignment: .position)
+```
+
+The `.frame()` modifier method has the following optional parameters:
+
+- `x` is entered as an integer to define the view's width in `pt` (points). SwiftUI assigns the view's width to the frame if `x` is given `nil` as a value or `width: x` is left out.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/viewmodifier/terms/frame/frame.md:31]: Can we remove the line breaks between the bullet points.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla I did a first review and had one comment.

File: content/swiftui/concepts/viewmodifier/terms/frame/frame.md (hunk 1 above)
Comment (cslylla) [reply content/swiftui/concepts/viewmodifier/terms/frame/frame.md:31]: I removed them.

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good for review two.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@cslylla I've conducted a second review and added a few minor changes to this.

**Comment by KTom101** [comment]
@cslylla Thanks for making the changes!

@SSwiniarski Does this look good to merge?

**Review by KTom101 — APPROVED** [review]

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good to me.

**Comment by SSwiniarski** [comment]
@cslylla Congrats on your contribution! You can find the entry here:https://www.codecademy.com/resources/docs/swiftui/viewmodifier/frame

image

Looking forward to your next contribution.

**Pull request merged**